README.Rmd

```diff
@@ -22,72 +22,82 @@ knitr::opts_chunk$set(
 [![R-CMD-check](https://github.com/shawnlaffan/Biodiverse-R/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/shawnlaffan/Biodiverse-R/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-Provides an R interface to the spatial analyses available in Biodiverse. Biodiverse 
+Provides an R interface to the analyses available in Biodiverse. Biodiverse 
 is a tool for the spatial analysis of diversity using indices based on taxonomic, 
 phylogenetic, trait and matrix-based (e.g. genetic distance) relationships, 
 as well as related environmental and temporal variations. More information is 
 available at its [Github page](https://github.com/shawnlaffan/biodiverse).
 
 ## Installation
 
-This is a two step process.  First, install the Biodiverse engine.  Second, install the R package.  
+This is a two step process.  First, install Perl so the 
+Biodiverse engine can run.  Second, install the BiodiverseR R package.  
 
-1. Install the Biodiverse engine. 
+1. Install Perl 
 
-This currently requires a working perl interpreter in your path.
+BiodiverseR currently requires a working perl interpreter in your path.
 (Future versions will provide self contained executables).
 
-On Windows a perl interpreter can be obtained through the [Strawberry perl project](https://strawberryperl.com/releases.html).
-Most unix-derived systems provide a perl interpreter but it is best to avoid this and use
-a system like [perlbrew](https://perlbrew.pl/).  Be sure to also install the cpanm utility (see
-perlbrew site for details).  
+On Windows a perl interpreter can be obtained through the 
+[Strawberry perl project](https://strawberryperl.com/releases.html).
+This will be downloaded automatically when using the commands below.
 
-Once you have a perl installed and in your path you can install the perl dependencies using
-cpanm at the command line.  (Make sure to update the below code to use the correct path separator
-on Windows).  Note that this requires that you have git installed on your system and in the path.
+Most unix-derived systems such as Linux and Mac provide a perl interpreter 
+but it is best to avoid this and install 
+[perlbrew](https://perlbrew.pl/) so you have a separate installation.  
+When you install perlbrew be sure to also install the cpanm utility (see
+the perlbrew site for details).  
 
-This code also assumes you have run a git clone of this repo and are at the top level of this repo.
 
+You also need to have git installed on your system and in the path.
```

Comment:
Coming from a newbie with no CS background, what does "in the path" mean? How can I check? 😂  I would get stuck, and I think a typical BiodiverseR would probably have limited CS knowledge.

Reply:
Could probably delete "in the path" given it will be once installed, unless the user has deliberately disabled it, in which case they are less likely to be a newbie.

Reply:
Removed now.